sql-plugin/src/main/scala/com/nvidia/spark/rapids/SortUtils.scala

```diff
@@ -261,20 +261,27 @@ class GpuSorter(
               // It is slower, but it works
               val concatenated = closeOnExcept(tablesToMerge) { _ =>
                 while (spillableBatches.nonEmpty) {
-                  withResource(spillableBatches.pop()) { sb =>
+                  // We retry work with a single spillable batch, then it can be closed
```

Comment:
So do you plan on having a follow on issue to fix this?

If we need to make tablesToMerge Spillable we should do that.

Reply:
Sorry for cunfusion, and it is no longer an issue. Please ignore it.